src/contrib/cppheaderparser/CppHeaderParser/CppHeaderParser.py

```diff
@@ -1879,7 +1879,7 @@ def evaluate_property_stack(self):
                 typedef = self._parse_typedef( self.stack )
                 name = typedef['name']
                 klass = self.classes[ self.curClass ]
-                klass[ 'typedefs' ][ self.curAccessSpecifier ].append( name )
+                klass[ 'typedefs' ][ self.curAccessSpecifier ].append( typedef )
```

Comment:
@tpaviot sure, I'm from Cuba, I don't know the reason, the site just keeps loading.